pmd-java/src/test/java/net/sourceforge/pmd/typeresolution/TypeInferenceTest.java

```diff
@@ -34,49 +34,59 @@
     private Variable b = new Variable();
 
     @Test
-    public void testEqualityReduce() {
-
+    public void testEqualityReduce1() {
```

Comment:
can we try better names? Maybe something such as `testEqualityReduceProperVsProper()`?